### PR TITLE
Remove sequential support from DPC++ examples

### PR DESCRIPTION
**Pull request by napetrov**
# Description
remove sequential support from male examples and win solution

### CODE REVIEW

**Comment by napetrov** [comment]
/intelci: run
set infra_branch=dev/napetrov_cleanup

**Comment by napetrov** [comment]
/intelci: run
set infra_branch=dev/napetrov_cleanup

**Comment by napetrov** [comment]
/intelci: restart
set infra_branch=dev/napetrov_cleanup

**Comment by napetrov** [comment]
/intelci: restart
set infra_branch=dev/napetrov_cleanup

**Comment by napetrov** [comment]
Restarted ci for windows
http://intel-ci.intel.com/eb11e11b-a9f9-f166-9ec3-8cdcd4b723a1

File: examples/daal/cpp_sycl/makefile_win [hunk 1]
```diff
@@ -102,7 +100,7 @@ COPTS=-m64 $(COPTS)
 
 !IF ("$(RES_EXT)"=="lib")
 !IF ("$(threading)"=="sequential")
-DAAL_LIB_T = "$(DAAL_PATH)\onedal_sequential.lib"
+$(error Sequential version is not supported)
```
Comment (rlnx) [inline examples/daal/cpp_sycl/makefile_win:103]: Does this work with Windows nmake?

Comment (rlnx) [reply examples/daal/cpp_sycl/makefile_win:103]: I have concern about message. It sounds like it's not supported by DPC++ interfaces at all, it's not truth. 

"Sequential version is not supported. To run sequential version please refer to 'examples/daal/cpp'"

**Comment by napetrov** [comment]
/intelci: run
set infra_branch=dev/napetrov_cleanup

File: examples/daal/cpp_sycl/makefile_win (hunk 1 above)
Comment (napetrov) [reply examples/daal/cpp_sycl/makefile_win]: fixed

File: examples/daal/cpp_sycl/makefile_lnx [hunk 2]
```diff
@@ -87,8 +84,7 @@ else
 endif
 
 ifeq ($(threading),sequential)
-    DAAL_LIB_T := $(DAAL_LIB_EXT),-lonedal_sequential
-    EXT_LIB += $(addprefix -L,$(TBB_PATH)) -ltbb -ltbbmalloc
+    $(error Sequential version is not supported)
```
Comment (rlnx) [inline examples/daal/cpp_sycl/makefile_lnx:87]: The message need to be aligned with `makefile_win`

Comment (napetrov) [reply examples/daal/cpp_sycl/makefile_lnx]: fixed

File: examples/oneapi/dpc/makefile_lnx [hunk 3]
```diff
@@ -86,7 +83,7 @@ else
 endif
 
 ifeq ($(threading),sequential)
-    THREADING_LIB := onedal_sequential
+    $(error Sequential version is not supported)
```
Comment (rlnx) [inline examples/oneapi/dpc/makefile_lnx:86]:
```suggestion
     $(error Sequential version is not supported. To run sequential version please refer to 'examples/oneapi/cpp')
```

Comment (napetrov) [reply examples/oneapi/dpc/makefile_lnx]: yes. missed it last time . thanks

File: examples/oneapi/dpc/makefile_lnx [hunk 4]
```diff
@@ -86,7 +83,7 @@ else
 endif
 
 ifeq ($(threading),sequential)
-    THREADING_LIB := onedal_sequential
+    $(error Sequential version is not supported. To run sequential version please refer to 'examples/daal/cpp')
```
Comment (rlnx) [inline examples/oneapi/dpc/makefile_lnx:86]: That's wrong message, I've suggested the different one. refer to `examples/oneapi/cpp`

Comment (napetrov) [reply examples/oneapi/dpc/makefile_lnx]: done

**Review by rlnx — APPROVED** [review]

**Pull request merged**